assets/js/scripts.js

```diff
@@ -27,25 +31,21 @@
   }
 
   async function projectGridSetup() {
-    // Set the path and onProjectPage variables to help render the correct img.src and href values in the HTML, and to pass on to shuffleArray fn.
-    let path = window.location.pathname;
-    let onProjectPage = path.includes('revivals'); // ie, revival/reece/
-
     // Get data
     let fontData = await getJSONData();
-    let shuffledData = shuffleArray(fontData, onProjectPage, path);
+    let shuffledData = shuffleArray(fontData, onHomepage, path);
 
     // Render the project grid
     const parent = document.getElementById("projectGrid");
     for (let i = 0; i < shuffledData.length; i++) {
       const project = shuffledData[i];
 
       const singleProject = document.createElement("a");
-      singleProject.href = !onProjectPage ? project.url : `./${project.url}`;
+      singleProject.href = onHomepage ? `.${project.url}` : `./..${project.url}`;
```

Comment:
Again, different paths depending on whether we are on the homepage or not. So these would appear in production:
<img width="541" alt="Screenshot_2023-07-27_at_10_13_55_AM" src="https://github.com/norawarschewski/tw-2023/assets/10406456/eadeb675-e5dc-4151-af6a-6692522d392a">